flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -396,7 +396,7 @@ static int32_t vtol_follower_control_accel(float dT)
 static float vtol_follower_control_altitude(float downCommand) {
 	AltitudeHoldStateData altitudeHoldState;
 	altitudeHoldState.VelocityDesired = downCommand;
-	altitudeHoldState.Integral = vtol_pids[DOWN_VELOCITY].iAccumulator / 1000.0f;
```

Comment:
I'm glad you thought of these.  Hopefully they're the only points we micromanage integrals.